frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js

```diff
@@ -13,7 +13,7 @@ const Approved = ({
   const managerNotesState = getEditorState(managerNotes || 'No manager notes');
 
   return (
-    <>
+    <div className="smart-hub--hide-notes">
```

Comment:
Not sure if there is a cleaner way to add this class. Adding the class name to the Fragment didn't work.